rllib/algorithms/callbacks.py

```diff
@@ -406,21 +382,21 @@ def on_episode_end(
         been converted to numpy arrays yet).
 
         Args:
-            episode: The terminated/truncated episode. On the new API stack, this will
-                be a SingleAgentEpisode or MultiAgentEpisode object (which have already
-                been finalized, meaning their data has been converted to numpy arrays).
-                On the old API stack, this will be a Episode or EpisodeV2 object.
-            env_runner: Replaces `worker` arg. Reference to the current EnvRunner.
-            env: Replaces `base_env` arg.  The gym.Env (new API stack) or RLlib
-                BaseEnv (old API stack) running the episode. On the old stack, the
-                underlying sub environment objects can be retrieved by calling
-                `base_env.get_sub_environments()`.
-            rl_module: Replaces `policies` arg. Either the RLModule (new API stack) or a
-                dict mapping policy IDs to policy objects (old stack). In single agent
-                mode there will only be a single policy/RLModule under the
-                `rl_module["default_policy"]` key.
-            env_index: The index of the sub-environment that is about to be reset
-                (within the vector of sub-environments of the BaseEnv).
+            episode: The terminated/truncated SingleAgent- or MultiAgentEpisode object
```

Comment:
Dumb question: Can a user then also "rewrite an episode"? This could make sense if for example videos or images should be logged and they come either within the `render_images` or the `info`s. We do not have to carry them along anymore in the episodes ... makes it way more memory-efficient.